apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/valueset/ValueSetReader.java

```diff
@@ -99,6 +99,7 @@ public Concept toConcept(CodeValueGeneral codeValueGeneral) {
                 codeValueGeneral.getConceptPreferredNm(),
                 codeValueGeneral.getCodeSystemDescTxt(),
                 codeValueGeneral.getConceptStatusCd(),
+                codeValueGeneral.getConceptCode(),
```

Comment:
It looks like `getConceptCode()` is already being called on line 98, and corresponds to the `value` field in the `Concept` object.

Reply:
@mpeels  sure I removed the additional call to getConceptCode() but renamed value => conceptCode in the response for better readability.